internal/config/default/config.toml

```diff
@@ -132,7 +132,7 @@ limit = 0
   revision_command = ["show", "--color", "always", "-r", "$change_id"]
   oplog_command = ["op", "show", "$operation_id", "--color", "always"]
   file_command = ["diff", "--color", "always", "-r", "$change_id", "$file"]
-  show_at_bottom = false
+  position = "auto"
```

Comment:
How can people know possible valid values for position? Remember that most people don't tend to read documentation, perhaps adding a comment of possible values is enough ?

Reply:
@vic updated

Though log.Fatal does not show in stdout stderr by default. One has to define DEBUG=1 and look in debug.log https://github.com/lbonn/jjui/blob/6e95d19846d33eb0fbd07d9563a2ad02cd9f9136/cmd/jjui/main.go#L118